compiler/resolution/wrappers.cpp

```diff
@@ -477,17 +477,19 @@ buildDefaultWrapper(FnSymbol* fn,
 
 FnSymbol*
 defaultWrap(FnSymbol* fn,
-            Vec<ArgSymbol*>* actualFormals,
+            std::map<int, ArgSymbol*>* actualFormals,
             CallInfo* info) {
   FnSymbol* wrapper = fn;
-  int num_actuals = actualFormals->n;
+  int num_actuals = actualFormals->size();
   int num_formals = fn->numFormals();
   if (num_formals > num_actuals) {
     Vec<Symbol*> defaults;
```

Comment:
Seems a shame not to take the opportunity to upgrade the variable defaults to be a std::vector.
But may that has too much impact on addCache() and buildDefaultWrapper()?